rpm/xulrunner-qt5.spec

```diff
@@ -277,10 +304,45 @@ export SB2_RUST_USE_REAL_EXECVP=Yes
 export SB2_RUST_USE_REAL_FN=Yes
 %endif
 
+echo "export CC=gcc" >> "%BUILD_DIR"/rpm-shared.env
+echo "export CXX=g++" >> "%BUILD_DIR"/rpm-shared.env
+echo "export AR=\"gcc-ar\"" >> "%BUILD_DIR"/rpm-shared.env
+echo "export NM=\"gcc-nm\"" >> "%BUILD_DIR"/rpm-shared.env
+echo "export RANLIB=\"gcc-ranlib\"" >> "%BUILD_DIR"/rpm-shared.env
+
+# This avoids a malloc hang in sb2 gated calls to execvp/dup2/chdir
+# during fork/exec. It has no effect outside sb2 so doesn't hurt
+# native builds.
+echo "export SB2_RUST_EXECVP_SHIM=\"/usr/bin/env LD_PRELOAD=/usr/lib/libsb2/libsb2.so.1 /usr/bin/env\"" >> "%BUILD_DIR"/rpm-shared.env
+echo "export SB2_RUST_USE_REAL_EXECVP=Yes" >> "%BUILD_DIR"/rpm-shared.env
+echo "export SB2_RUST_USE_REAL_FN=Yes" >> "%BUILD_DIR"/rpm-shared.env
+
+echo "export CARGOFLAGS=\" --offline\"" >> "%BUILD_DIR"/rpm-shared.env
+echo "export CARGO_NET_OFFLINE=1" >> "%BUILD_DIR"/rpm-shared.env
+echo "export CARGO_BUILD_TARGET=armv7-unknown-linux-gnueabihf" >> "%BUILD_DIR"/rpm-shared.env
+echo "export CARGO_CFG_TARGET_ARCH=arm" >> "%BUILD_DIR"/rpm-shared.env
+
+%build
+
+source "%BUILD_DIR"/rpm-shared.env
+
+#%ifarch %arm
+echo "ac_add_options --host=armv7-unknown-linux-gnueabihf" >> "$MOZCONFIG"
+echo "ac_add_options --target=armv7-unknown-linux-gnueabihf" >> "$MOZCONFIG"
+#%endif
+
 # hack for when not using virtualenv
 ln -sf "%BUILD_DIR"/config.status $PWD/build/config.status
 
 %ifarch %arm
```

Comment:
Needs aarch64 too

Reply:
or better %arm64 and change %arm to %arm32 (see: https://github.com/rpm-software-management/rpm/blob/rpm-4.16.x/macros.in#L1137).

Reply:
I've added in aarch64 here and also gone through everything so it's now using `%arm32` and `%arm64`.